kong/resty/dns_client/init.lua

```diff
@@ -0,0 +1,635 @@
+-- vim: ts=4 sts=4 sw=4 et:
+
+local utils = require("kong.resty.dns_client.utils")
+local mlcache = require("kong.resty.mlcache")
+local resolver = require("resty.dns.resolver")
+
+local now = ngx.now
+local log = ngx.log
+local WARN = ngx.WARN
+local ALERT = ngx.ALERT
+local math_min = math.min
+local timer_at = ngx.timer.at
+local table_insert = table.insert
+local ipv6_bracket = utils.ipv6_bracket
+-- local deep_copy = function (t) return t end -- TODO require("kong.tools.utils").deep_copy
+
+-- debug
```

Comment:
TODO: clean code & introduce appropriate DEBUG log instead of logt().

Reply:
done https://github.com/Kong/kong/pull/12305/commits/fe6332d4396b67096d4a40bd54e9bb7b697b7459